src/main/java/com/dreammaster/scripts/ScriptEFR.java

```diff
@@ -0,0 +1,69 @@
+package com.dreammaster.scripts;
+
+import static gregtech.api.enums.Mods.EtFuturumRequiem;
+import static gregtech.api.enums.Mods.Minecraft;
+import static gregtech.api.enums.Mods.Thaumcraft;
+import static gregtech.api.util.GTModHandler.getModItem;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.dreammaster.recipes.CustomItem;
+
+import gregtech.api.enums.ItemList;
+import gregtech.api.util.GTModHandler;
+
+public class ScriptEFR implements IScriptLoader {
+
+    @Override
+    public String getScriptName() {
+        return "Et Futurum Requiem";
+    }
+
+    @Override
+    public List<String> getDependencies() {
+        return Arrays.asList(EtFuturumRequiem.ID, Thaumcraft.ID);
+    }
+
+    @Override
+    public void loadRecipes() {
+        long bits = GTModHandler.RecipeBits.NOT_REMOVABLE | GTModHandler.RecipeBits.REVERSIBLE
+                | GTModHandler.RecipeBits.BUFFERED;
+        GTModHandler.addCraftingRecipe(
+                GTModHandler.getModItem(EtFuturumRequiem.ID, "observer", 1L),
+                bits,
+                new Object[] { "AEA", "BCD", "AAA", 'A', "cobblestone", 'B',
+                        GTModHandler.getModItem(Minecraft.ID, "comparator", 1, 0), 'C', ItemList.Sensor_LV, 'D',
```

Comment:
LV Sensor for an observer seems too expensive

Reply:
if at least instead of using cobblestone it used compressed cobble and it gives 8 or 16 ...

Reply:
we are not adding stuff like this with vanilla recipes and LV is hardly late. But I can have a look at our recipes for similar modded blocks and go more in that direction

Reply:
changed the recipe